### ISSUE TITLE
Investigate and fix sync issue related to Moped's URL and label in Knack

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#8868]
This issue is to document a task I have worked on today:

@johnclary brought this to the group's attention in our recent planning meeting and was kind enough to share more details with me later in the day. 🙏 

In production, the current configuration of production Data Tracker appears to differ from the development configuration. The Moped application is geared to use configuration as presented by the development Data Tracker, and this issue is intended to capture getting these aligned so the URL and label can be used in production Data Tracker.

Through investigation of the payloads sent back and forth between the Moped editor and the Knack endpoint, it's found that Moped is sending data as we intend. I have sent the following to John over slack:

> I think we need to adjust the “Link Text Format” configuration on the Link field to “Use Unique Texts for each link” to cause Knack to begin to accept the label K/V pair in the payload JSON. (FWIW, the payload out of moped looks like it’s just as we’d expect it to be.)  Also, I think we need to get rid of the Conditional Rules entry that sets the value of Moped URL into the Link.

I'll continue my conversation with him and execute his direction and document that progress here. Thanks all!

This issue may be followed by a forthcoming follow-on action that documents backfilling the label field into production Data Tracker for Moped Projects.

**Comment by frankhereford** [comment]
John requested that I double check that I can reproduce the exact behavior in dev by configuring it as is prod. While I believe I have conducted this test, I did not make an adequate recording of the results to show, so I am going to do it again with better documentation and leave that here. 

At this point, he suggested that I present the work done to @amenity and discuss what backfilling code may need be written to get this integrated into her plans and schedules as is most helpful for her.
